raiden/network/proxies/token_network.py

```diff
@@ -1226,7 +1228,7 @@ def unlock(
             checking_block,
             'unlock',
             channel_identifier,
-            self.node_address,
+            participant,
```

Comment:
Yes, this is basically because we can either try to unlock our side or the partner's side... see raiden_event_handler for how this function is called with the above params.